streams/th.m3u

```diff
@@ -76,6 +84,8 @@ https://thainews.prd.go.th/lv/live/ch1_L_L.sdp/playlist.m3u8
 http://news1.live14.com/stream/news1.m3u8
 #EXTINF:-1 tvg-id="PoliceTV.th",Police TV (1080p)
 https://cdn-th-vip.livestreaming.in.th/policetv/policetv/playlist.m3u8
+#EXTINF:-1 tvg-id="PPTVHD36.th",PPTV HD (720p)
+https://cdn6.goprimetime.info/feed/202306140918/chpptvhd3/index.m3u8
```

Comment:
```suggestion
```

"PPTV HD 36" is on the blocklist due to claims of copyright holders: https://github.com/github/dmca/blob/master/2020/09/2020-09-16-dfl.md